tests/mock/models.py

```diff
@@ -96,6 +97,9 @@ def create_service_with_dependencies(include_execution=False,
     if include_input:
         input = create_parameter(name='input1', value='value1')
         service.inputs = {'input1': input}
+    if include_output:
```

Comment:
I think this should be done separately -- I'm trying to make this commit specifically about the CLI.